build.sbt

```diff
@@ -1,3 +1,7 @@
+lazy val scala211Version = "2.11.12"
+lazy val scala212Version = "2.12.10"
+lazy val scala213Version = "2.13.0"
```

Comment:
2.13.1

Reply:
Currently, kind-projector doesn't support 2.13.1. 
Issue: https://github.com/typelevel/kind-projector/issues/116.

Should we wait for their release first?